rules/order.js

```diff
@@ -15,13 +18,35 @@ module.exports = {
         'right',
         'bottom',
         'left',
+        'visibility',
+        'opacity',
       ],
     },
     {
       order: 'strict',
       properties: [
         'transform',
         'transform-origin',
+        // Animating transforms is common, so let's group them together
```

Comment:
I was thinking opacity could be in this group as well.

Not sure we should put them at the bottom though. Transforms have a big impact on the positioning and in Concentric CSS fashion it would probably make sense to have those at the top.